opennms-webapp-rest/src/main/java/org/opennms/web/rest/support/SearchProperties.java

```diff
@@ -501,6 +503,25 @@
 		IF_SERVICE_SERVICE_PROPERTIES.addAll(withAliasPrefix(Aliases.serviceType, "Service", SERVICE_TYPE_PROPERTIES));
 		IF_SERVICE_SERVICE_PROPERTIES.addAll(withAliasPrefix(Aliases.snmpInterface, "SNMP Interface", SNMP_INTERFACE_PROPERTIES));
 
+		// Root prefix
+		IP_INTERFACE_SERVICE_PROPERTIES.addAll(IP_INTERFACE_PROPERTIES);
+		IP_INTERFACE_SERVICE_PROPERTIES.addAll(withAliasPrefix(Aliases.node, "Node", NODE_PROPERTIES));
+		IP_INTERFACE_SERVICE_PROPERTIES.addAll(withAliasPrefix(Aliases.ipInterface, "IP Interface", IP_INTERFACE_PROPERTIES));
+		//IP_INTERFACE_SERVICE_PROPERTIES.addAll(withAliasPrefix(Aliases.monitoredService, "Monitored Service", IF_SERVICE_PROPERTIES));
```

Comment:
is that intended to be commented out?

Reply:
It's a copy of the sub-resources from the node properties, I was assuming since it was commented out there's something that doesn't work with it.